vitest.preset.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference types='vitest' />
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    globals: true,
+    watch: false,
+    include: ['src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts}'],
+    reporters: ['default'],
+    coverage: {
+      reporter: ['text', 'lcov'],
+      exclude: ['mocks/**', '**/types.ts'],
+      provider: 'v8',
+    },
```

Comment:
Running with `--coverage.enabled` requires installing `@vitest/coverage-v8` dev dependency:

![image](https://github.com/user-attachments/assets/845c434d-e880-4ee2-9509-2f55bc8f9d8d)

Reply:
Oops, forgot that, thank you.